src/typography/typography.ts

```diff
@@ -0,0 +1,199 @@
+import { Colors } from "./colors";
+
+export const constants = {
+  font: {
+    family: {
+      primary: "Poppins",
+    },
+    lineHeight: {
+      small: "40px",
+      medium: "48px",
+      large: "60px",
+    },
+    size: {
+      xxxxLarge: {
+        option1: "40px",
+        option2: "60px",

```

Comment:
isso ta mesmo confuso no figma, mas eu usaria apenas uma das opcoes sempre, ou seja, desconsideraria a opcao 2 e atribuiiria direto a 1 em todos os tamanhos:

```suggestion
      xxxxLarge: "40px",
```

O que acham @alanraso @Japoncio3k ?

Reply:
Eu concordo, ou utilizaria opção 1 em todas as propriedades ou a opção 2 em todas as propriedades, o importante é seguir o padrão.